cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgsfourleptons.json

```diff
@@ -0,0 +1,92 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Wunsch, Stefan",
+        "orcid": "0000-0002-4485-2972"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_published": "2021",
+    "distribution": {
+      "formats": [
+        "cc",
+        "py",
+        "zip"
+      ],
+      "number_files": 4,
+      "size": 999
+    },
+    "doi": "10.7483/OPENDATA.CMS.TODO",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:999",
+        "size": 999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/software/TODO/HiggsToFourLeptonsNanoAODOutreachAnalysis-1.0.zip"
+      }
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12360",
+    "run_period": [
+      "Run2012A",
+      "Run2012B"
+    ],
+    "source_code_repository": {
+      "url": "www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis"
+    },
+    "system_details": {
+      "description": "This code can be used with the CMS Open Data VM environment.",
+      "release": "ROOT 6.22 or later"
+    },
+    "title": "Analysis of Higgs boson decays to four leptons using data and simulation of events at the CMS detector from 2012",
```

Comment:
We are reviewing the authorship practices and going towards the direction that CMSSW-based SW records would have "CMS open data group" as author. This record is different as it is not CMSSW-dependent and is an implementation of this analysis in the ROOT, using ROOT dataframe. To justify the single author (in this case very justified), it would be good to have that explicitly in the title (with the risk of it becoming loooooonnnnngggg...)

It could be "Implementation using ROOT dataframes of analysis of ...."

@caredg what do you think?

Reply:
I agree, this is one example where an specific author is justified.  The title is going to be long anyways, but one way to shorten it would be to add just "using ROOT's RDataFrame".

Reply:
That's better indeed. We can check with @tiborsimko if it is possible to add this i.e. ("using ROOT's RDataFrame") to the titles of the two other examples as well (ie. recids 12342 and 212350). But in any case, we should go for this title for this record.

Reply:
I've appended `using ROOT's RDataFrame` to the title!